src/DefaultBuilder/src/WebApplication.cs

```diff
@@ -86,15 +86,23 @@ IServiceProvider IApplicationBuilder.ApplicationServices
         /// <param name="args">Command line arguments</param>
         /// <returns>The <see cref="WebApplication"/>.</returns>
         public static WebApplication Create(string[]? args = null) =>
-            new WebApplicationBuilder(Assembly.GetCallingAssembly(), args).Build();
+            new WebApplicationBuilder(Assembly.GetCallingAssembly(), new() { Args = args }).Build();
```

Comment:
Y NO CHAINING? `Create(string[])` -> `CreateBuilder(string[])` + `Build()` -> `CreateBuilder(WebApplicationOptions)`

Reply:
Because of `Assembly.GetCallingAssembly()` (which will be removed in another change)

Reply:
Aaaah 💡